autopush-common/Cargo.toml

```diff
@@ -34,11 +34,11 @@ openssl = "0.10"
 rand = "0.8"
 regex = "1.4"
 # XXX: pin to < 0.10 until futures 0.3
-reqwest = "0.9.24"
+reqwest = "0.11"
```

Comment:
Looks like either the comment is outdated or the version shouldn't be upgraded?

Reply:
Ah, thanks! Missed that comment, I'll make it a bit clearer for myself.